server/aws-lsp-codewhisperer/src/language-server/testUtils.ts

```diff
@@ -9,6 +9,8 @@ export const HELLO_WORLD_IN_CSHARP = `class HelloWorld
     }
 }`
 
+export const SPECIAL_CHARACTER_HELLO_WORLD = `{class HelloWorld`
```

Comment:
how do you know this string is below the threshold?

Reply:
Reviewed the criteria for the calculation and tested this out by printing the value to verify while testing locally  - it is below threshold.